README.md

```diff
@@ -1,42 +1,66 @@
-# Foxglove Studio WebSocket protocol libraries
+# Foxglove WebSocket Protocol
 
-This repository provides a protocol specification and reference implementations enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
+This repository provides a [protocol specification](docs/spec.md) and reference implementations enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
 
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The protocol is encoding-agnostic, as long as the desired encoding is supported by both client and server. It currently supports Protobuf and JSON messages.
 
-The following implementations are provided in this repository and as installable packages:
+The following implementations are available as installable packages:
 
 | Language              | Includes         | Package name                     | Version                                                                                                                      |
 | --------------------- | ---------------- | -------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| Python                | server           | `foxglove-websocket`             | [![](https://shields.io/pypi/v/foxglove-websocket)](https://pypi.org/project/foxglove-websocket/)                            |
-| JavaScript/TypeScript | server + client  | `@foxglove/ws-protocol`          | [![](https://shields.io/npm/v/@foxglove/ws-protocol)](https://www.npmjs.com/package/@foxglove/ws-protocol)                   |
-| JavaScript/TypeScript | examples         | `@foxglove/ws-protocol-examples` | [![](https://shields.io/npm/v/@foxglove/ws-protocol-examples)](https://www.npmjs.com/package/@foxglove/ws-protocol-examples) |
+| Python                | [server](python)            | `foxglove_websocket`             | [![](https://shields.io/pypi/v/foxglove-websocket)](https://pypi.org/project/foxglove-websocket/)                            |
+| JavaScript/TypeScript | [server + client](typescript/ws-protocol)  | `@foxglove/ws-protocol`          | [![](https://shields.io/npm/v/@foxglove/ws-protocol)](https://www.npmjs.com/package/@foxglove/ws-protocol)                   |
+|   | [examples](typescript/ws-protocol-examples)         | `@foxglove/ws-protocol-examples` | [![](https://shields.io/npm/v/@foxglove/ws-protocol-examples)](https://www.npmjs.com/package/@foxglove/ws-protocol-examples) |
 | C++                   | _in development_ |
 
-## Documentation
+## Python server
```

Comment:
I'm not convinced we should have all the Python and JS/TS examples in this main README. As we add more languages it could get pretty crowded, and it makes it a bit more difficult to maintain the readmes since we need to update both places. You didn't like just linking out to the docs for individual packages from here?